crates/bevy_debug_draw/src/debug_draw.rs

```diff
@@ -0,0 +1,200 @@
+use std::f32::consts::TAU;
+
+use bevy_asset::Handle;
+use bevy_ecs::system::Resource;
+use bevy_math::{vec3, Quat, Vec2, Vec3};
+use bevy_render::prelude::{Color, Mesh};
+
+/// A resource with an immediate mode API for drawing lines and wireshapes.
+/// Useful for visual debugging.
```

Comment:
Docs should openly specify how long the debug lines last on screen. One frame? Multiple?

Reply:
An immediate mode drawing API is frame by frame, but I'll do a docs pass and write it down in plain language :)